docs/ops/type/ConvertLike_1.md

```diff
@@ -2,37 +2,35 @@
 
 **Versioned name**: *ConvertLike-1*
 
-**Category**: type conversion
+**Category**: *Type conversion*
 
-**Short description**: Operation converts all elements of the 1st input tensor to a type of elements of 2nd input tensor.
+**Short description**: *ConvertLike* operation performs element-wise conversion on a given input tensor `data` to the element type of an additional input tensor `like`.
 
-**Attributes**:
+**Detailed description**
 
-    No attributes available.
+Conversion from one supported type to another supported type is always allowed. User must be aware of precision loss and value change caused by range difference between two types. For example, a 32-bit float *3.141592* may be round to a 32-bit int *3*. The result of unsupported conversions is undefined, e.g. convertion of negative signed integer value to any unsigned integer type.
```

Comment:
`convertion` -> `conversion`

Reply:
Done. Please notice that I also corrected this from Convert as we missed it in the previous review